app/main.py

```diff
@@ -1 +1,46 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    # @classmethod
```

Comment:
i can see its just a comment, but i am not sure its necessary to use it here

Reply:
fixed